File: test/bench/for/for.js
const start = Date.now()

let list = []

for (let i = 0; i < 1000000; i += 1) {
    list.push(i)
}

let sum = 0
for (let i of list) {
    sum += i
}

console.log(`time: ${Date.now() - start}`)
console.log(sum)

File: src/test/bench/for/for.lua
local start = os.clock()
local list = {}
for i = 0, 999999 do
  list[i] = i
end

local sum = 0
for k, i in pairs(list) do
  sum = sum + i
end

io.write("time: ", (os.clock() - start) * 1000, "\n")
io.write(sum .. "\n")